pkg/manager/member/pd_member_manager.go

```diff
@@ -951,6 +968,15 @@ func (m *pdMemberManager) collectUnjoinedMembers(tc *v1alpha1.TidbCluster, set *
 	return nil
 }
 
+// TODO: Support check status http request in future.
+func buildPDReadinessProbHandler() corev1.Handler {
```

Comment:
Can we include `*v1alpha1.TidbCluster` as an input parameter here although it's not in use for now? Just like for `TiDB`, we might need to construct some non-default handler based on the `TidbCluster` spec later.